models/user.js

```diff
@@ -138,14 +138,17 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.INTEGER,
       allowNull: false,
       defaultValue: DefaultValuesConstants.USER_UNAUTHORIZED_LEAVES
+    },
+    teamId: {
+      type: DataTypes.INTEGER
     }
   }, { paranoid: true });
 
   User.associate = (models) => {
     User.hasMany(models.Otp, { as: 'otps', foreignKey: 'userId', foreignKeyConstraint: true });
     User.hasMany(models.Leave, { as: 'leaves', foreignKey: 'userId', foreignKeyConstraint: true });
     User.belongsTo(models.Team, { as: 'team', foreignKey: 'teamId', foreignKeyConstraint: true });
-    User.belongsTo(models.Designation, { as: 'designation', foreignKey: 'designation', foreignKeyConstraint: true });
+    User.belongsTo(models.Designation, { as: 'Designation', foreignKey: 'designation', foreignKeyConstraint: true });
```

Comment:
@sijusamson This will include the _designation_ model as **Designation** instead of **designation** :thinking:

Reply:
Is it okay If i do like this ?
` User.belongsTo(models.Designation, { as: 'designations', foreignKey: 'designation', foreignKeyConstraint: true });`

Reply:
Updated :+1: